main/src/test/java/org/mobilitydata/gtfsvalidator/validator/RouteNameValidatorTest.java

```diff
@@ -122,4 +123,20 @@ public void equalRouteShortNameRouteLongNameAndRouteDescShouldGenerateTwoNotices
         .contains(
             new SameNameAndDescriptionForRouteNotice(1, "r1", "duplicate", "route_short_name"));
   }
+
+  @Test
+  public void nullOrEmptyRouteLongNameShouldGenerateNotice() {
+    assertThat(validateRoute(createRoute("short name", null, "desc")))
```

Comment:
Isn't it the same test as in line 103?

Reply:
Indeed, collapsed these tests in https://github.com/MobilityData/gtfs-validator/pull/797/commits/acda7f672e3e0bb58167d297b5f6e3ee7e9c8367.